great-idea/css/index.css

```diff
@@ -63,4 +63,186 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+/* * {
+	box-sizing: border-box;
+} */
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+body{
+	width: 800px;
+	margin: 0 auto;
+}
+
+nav a {
+	padding: 20px;
+	text-decoration: none;
+	opacity: .3;
+	color: black;
+	font-size: 12px;
+}
+
+nav {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	width: 80%;
+}
+
+.header-container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.logo {
+	height: 30px;
+}
+
+h1 {
+	font-size: 80px;
+	text-align: center;
+	
+}
+
+header { 
+	/* display: flex; */
+	border-bottom: 2px solid black;
+}
+
+
+
+.action-bar {
+	display:flex;
+	padding: 50px 0px;
+	justify-content: space-between;
+	
+
+}
+
+/* .header-image {
+	display: flex;
+	justify-content: flex-end;
+} */
+
+.on-demand {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	width: 50%;
+	text-align: center;
+	align-items: center;
+	
+}
+
+button{
+	border: 1px solid black;
+	width: 150px;
+	height: 40px;
+	padding: 8px 34px;
+}
+
+.toptext {
+	display: flex;
+	justify-content: flex-start;
+	padding: 20px 0px;
+}
+
+.bottomtext {
+	display: flex;
+	justify-content: space-around;
+	margin:40px 0;
+}
+
+.middle-img {
+	width: 99%;
+}
+
+footer address{
+	border-top: 2px solid black;
+	padding: 30px 0px;
+}
+
+.copyright {
+	text-align: center;
+}
+
+.services-header-img {
+	width: 99%;
+}
+
+h3 {
+	font-size: 30px;
+
+}
+
+.services {
+	padding: 30px 0;
+}
+
+.center-fold {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	align-items: flex-start;
+	padding: 30px 0px 30px 0px;
+	border-bottom: 2px solid black;
+}
+
+.center-child {
+	border: 2px solid black;
+	/* width: 49%; */
+	display: flex;
+	flex-wrap: wrap;
+	margin: 10px 0px;
+	height: 240px;
+	padding: 20px;
+	width: 47%;
```

Comment:
Percent widths are important for responsive design so good job getting started using them now